### PR TITLE
Fix pandas_bridge and Travis CI 

### PR DESCRIPTION
**Pull request by alperyeg**
This PR changes the `install.sh` and `travis.yml` filles to use the newest libraries (numpy, scipy, pandas) to avoid hard-coding the version numbers and problems with interoperability between the different library versions. 
Additionally I fixed errors in `test_pandas_bridge`.  In the newest(?) panda version `0.16.2` the parameter `names` is also checked, when using `assert_index_equal ` in the unittests. This throws an error (as described here https://github.com/NeuralEnsemble/elephant/issues/33), because in our case e.g., the strings  `t_start` and `t_stop` are checked for equality.  So I set the parameter `names` to `False` not to check for equality.

### CODE REVIEW

File: elephant/test/test_pandas_bridge.py [hunk 1]
```diff
@@ -235,23 +235,23 @@ def test__spiketrain_to_dataframe__parents_empty(self):
         self.assertEqual(keys, res8.columns.names)
 
         for value, level in zip(values, res0.columns.levels):
-            assert_index_equal(value, level)
+            assert_index_equal(value, level, False, False)
```
Comment (apdavison) [inline elephant/test/test_pandas_bridge.py]: (1) please can you add the keywords i.e. `exact=False, check_names=False` to make it clearer what is going on.
(2) is it necessary to have `exact=False`?

**Comment by apdavison** [comment]
One of the things I like about the test matrix is that it tests both recent and old versions of the main dependencies. It's fine to update the most recent version to 1.9, and maybe we can update the "old" version to whatever is used in the most recent Ubuntu LTS release, but removing the hard-coded version numbers altogether makes Elephant more fragile.

File: elephant/test/test_pandas_bridge.py (hunk 1 above)
Comment (alperyeg) [inline elephant/test/test_pandas_bridge.py]: I will refactor this. `exact=False` may be set by default already. I have to check.

**Comment by alperyeg** [comment]
Travis uses Ubuntu 12.04, so we have to use the versions supported by 12.04. 
The problem I had was when installing the `pandas` library. If installed per pip it will automatically install the newest numpy. We can try to install an older version of pandas, at least for the Ubuntu distribution. 

I would suggest then not to use pip in the test matrix with the ubuntu distribution but its own package manager, because it seems that this is the most fragile test case.

**Comment by alperyeg** [comment]
I am closing this and will make a new clean PR.

**Pull request closed without merging**